NitroxServer/GameLogic/Entities/EntitySimulator.cs

```diff
@@ -78,7 +89,26 @@ private void AssignEntitiesToNewPlayers(Player oldPlayer, List<Entity> entities,
                     {
                         Log.Info("Player " + player.Name + " has taken over simulating " + entity.Id);
                         ownershipChanges.Add(new SimulatedEntity(entity.Id, player.Id, true, DEFAULT_ENTITY_SIMULATION_LOCKTYPE));
-                        return;
+                        break;
+                    }
+                }
+            }
+        }
+
+        private void AssignSimulationsToNewPlayers(Player oldPlayer, List<NitroxId> entities, List<SimulatedEntity> ownershipChanges)
```

Comment:
Is the old method `AssignEntitiesToNewPlayers` still used?  I don't see any references (though github is hard to navigate code in the review tool)